content/departments/product-engineering/process/planning.md

```diff
@@ -0,0 +1,93 @@
+
+Recommended reading:
+
+- Short article: [The perils of bad strategy](https://www.mckinsey.com/business-functions/strategy-and-corporate-finance/our-insights/the-perils-of-bad-strategy)
+- Book: [Good Strategy/Bad Strategy: The difference and why it matters](https://www.amazon.com/Good-Strategy-Bad-difference-matters/dp/1781256179/ref=sr_1_1?crid=1H0B1F3WWUL2S&keywords=good+strategy+bad+strategy&qid=1639180521&sprefix=good+strategy%2Caps%2C205&sr=8-1) by Richard Rumelt
+
+### Good plans are clear and concise
+
+A good plan contains:
+
+- A definition of success that:
+  - Is clear and succinct (so a variety of stakeholders can quickly understand what is important)
+  - Is solution agnostic (so the team has ownership and space to find and pursue the best solution)
+  - Is framed as a customer and/or business problem statement (so we are always working on the right things for our business)
+  - Can be unambiguously measured and evaluated (so there is no disagreement about whether we have achieved success)
+- A concrete set of deliverables we believe will solve the problem (so we can detect/manage dependencies and do capacity planning)
+
+This definition applies recursively and fractically at every layer of a plan (for example: company plan, product/eng department plan, org plan, team plan) for a given unit of time (year, quarter, week).
+
+### Planning is iterative
+
+Arriving at a good plan is an iterative process that is neither exclusively top-down nor exclusively bottom-up because there is a cyclical information dependency. Company level strategy is informed by what is working and not working about our business (context our teams have), and team plans are informed by what our company level strategy is (a decision ultimately owned by the CEO). Fundmentally, everyone on our team has context and insights that are valuable inputs into the planning process.
+
+In practice, both the top-down motion and bottoms-up motion happen concurrently, but we need to bootstrap the process to iteratively work through the information dependency. It is helpful to think about planning as having a `W` shape:
+![Planning shape](https://assets.proof.pub/2056/firstround/O6rbG1rNQ2emg8EPj1Xa_TKImage%200%20Leadership%20Frames.png)
+
+Source: [The Secret to a Great Planning Process — Lessons from Airbnb and Eventbrite](https://review.firstround.com/the-secret-to-a-great-planning-process-lessons-from-airbnb-and-eventbrite)
+
+### Building the delivery plan is part of the work
+
+Once a teams has a clear definition of success, there is usually work that needs to be done to figure out what the delivery plan is to achieve that success. The team might need to collect customer feedback, dig in to metrics, or build prototypes as a first step. This exploration is normal, healthy, and part of the delivery plan!
+
+### Plans are ambitious and feasible
```

Comment:
Can we add something here about what we think about how much time a person should be working on concrete deliverable plans vs. being in more flexible learning loops on their team? In Q4 we filled every team up with goals, with maybe 130% of their time dedicated to ambitiously delivering concrete goals and roadmap items, and it became very rigid and whenever we wanted to change any small thing it became a major replanning effort. Maybe this was intentional or maybe not, but it would help to write down what our idea is.

Reply:
Given the convo we had in directors team sync today, do you have updated thoughts on how this wording could be improved?

Reply:
I think it's fine as a first iteration, if we want to improve it I think we could say something about how fractally set plans that involve the same sets of people doing the work are meant to avoid overloading anyone in particular, or making things too rigid. I'm not sure there's a good answer yet, though, and this could probably come from our retrospective as an update to this page when we know more.

Reply:
I added some more context here